spec/services/notify_renewal_letter_service_spec.rb

```diff
@@ -3,24 +3,23 @@
 require "rails_helper"
 
 RSpec.describe NotifyRenewalLetterService do
-  describe "run" do
+  subject(:response) { described_class.run(registration: registration) }
+
+  describe ".run" do
     let(:registration) { create(:registration) }
-    let(:service) do
-      NotifyRenewalLetterService.run(registration: registration)
-    end
+    before { registration.registration_exemptions.update_all(expires_on: Date.new(2030, 7, 1)) }
 
     it "sends a letter" do
-      VCR.use_cassette("notify_renewal_letter") do
+      VCR.use_cassette("notify_renewal_letter", record: :new_episodes) do
```

Comment:
Minor point and not a blocker, but we shouldn't have to specify in an individual test that VCR can overwrite an existing cassette. The configuration might need some attention.

Reply:
ah yeah, this is no longer needed.